src/org/jetbrains/plugins/scala/project/Versions.scala

```diff
@@ -80,5 +99,11 @@ object Versions  {
       """.+>(\d+.\d+.+)/<.*""".r,
       Version("0.2.0"),
       Seq("0.2.0-RC1"))
+
+    val Hydra = Entity("https://repo.triplequote.com/artifactory/ivy-releases/com.triplequote/",
```

Comment:
One of our customers (maybe more than one) has mirrored our repository internally and their users don't have open access to the internet. So we should allow an easy way to change this URL, perhaps in the Hydra settings panel.

Reply:
@dragos I've discussed this point with Alex and it's hard to come up with a good UEX as we have so many URLs. What we agreed is that we will have two fields in the settings panel, one for providing the Maven mirror repo URL and another one for providing the Ivy mirror URL (in both cases, the user can provide multiple repos using comma). An alternative could be to have a panel similar to the one in Preferences > Maven > Repositories, but it's a bit more work, so I thought we can start simple and get the feedback from the IJ developers.

Another aspect that we should consider is transitive dependencies. For instance, Hydra depends on some logging libraries that may not be available in the set up mirrors. I'm not sure how we should handle this case (should we mirror the third-party dependencies we need in our own repos? or maybe shade the dependencies and create a fat jar for hydra?).